From a9f630a5fae6a5ea1eb346708184e15548fa8d5c Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Fri, 4 Mar 2022 19:28:39 -0600
Subject: [PATCH] Don't show extensions notice

Versions of gnome-tweaks prior to 40.x supported managing extensions, however this was removed in 40.0 and
a message stating that "Extensions support has moved" now pops up on the first launch of all versions >= 40.0.

Solus however has always disabled extensions support in gnome-tweaks and therefore our users gain nothing from 
seeing this message.

---
 data/org.gnome.tweaks.gschema.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/org.gnome.tweaks.gschema.xml b/data/org.gnome.tweaks.gschema.xml
index 1fe0237..bf6950a 100644
--- a/data/org.gnome.tweaks.gschema.xml
+++ b/data/org.gnome.tweaks.gschema.xml
@@ -2,7 +2,7 @@
 <schemalist>
     <schema path="/org/gnome/tweaks/" id="org.gnome.tweaks" gettext-domain="org.gnome.tweaks">
         <key type="b" name="show-extensions-notice">
-            <default>true</default>
+            <default>false</default>
             <summary>Show Extensions Notice</summary>
             <description>When first installed the user should be notified that extensions support has moved to a dedicated app, Extensions.</description>
         </key>
-- 
2.35.1

